python/tirex_tracker/__init__.py

```diff
@@ -691,6 +707,12 @@ def start(
         _handle_error(error_int)
         fetch_info_result = result_pointer.contents
 
+        # Start the CodeCarbon tracking, if available.
+        if CODECARBON_INSTALLED:
+            if codecarbon_tracker is None:
+                codecarbon_tracker = EmissionsTracker()
```

Comment:
Does this need an internet connection? There was, if I recall correctly, also an offline tracker? We would need the offline tracker here, so that things are directly compatible with tira, which would be very helpful :)